templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contributing
+
+---
+
+COMMENT TO REMOVE
+
+Be aware of XXX in sentences and links: they need to be replaced by the name of the actual repository.
+
+---
+
+First of all, thank you for contributing to MeiliSearch! The goal of this document is to provide everything you need to know in order to contribute to MeiliSearch and its different integrations.
+
+<!-- MarkdownTOC autolink="true" style="ordered" indent="   " -->
+
+- [Assumptions](#assumptions)
+- [How to Contribute](#how-to-contribute)
+- [Development Workflow](#development-workflow)
+- [Git Guidelines](#git-guidelines)
+
+<!-- /MarkdownTOC -->
+
+## Assumptions
+
+1. **You're familiar with [GitHub](https://github.com) and the [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)(PR) workflow.**
+2. **You've read the MeiliSearch [documentation](https://docs.meilisearch.com) and the [README](/README.md).**
+3. **You know about the [MeiliSearch community](https://docs.meilisearch.com/resources/contact.html). Please use this for help.**
+
+## How to Contribute
+
+1. Ensure your change has an issue! Find an [existing issue](https://github.com/meilisearch/XXX/issues/) or [open a new issue](https://github.com/meilisearch/XXX/issues/new). This is where you can get a feel if the change will be accepted or not.
```

Comment:
I don't really like the "This is where you can get a feel if the change will be accepted or not" because not only is not really encouraging to contribute, but also because I don't understand why this would give her/him a feeling about if it will be accepted :) Just my POV